### ISSUE TITLE
Mirador video annotation playback, wrong origin

### ISSUE THREAD

**Issue by daxid** [SCENE-CE/mirador-video#4]
Manifest URLs:  https://files.tetras-libre.fr/dev/vertical_video_with_annot.json

Open the above manifest URL in Mirador Video. It is a vertical video with an annotation targeted at `#xywh=0,0,400,400&t=0,12`

You can see the origin `0,0` used in the target correspond to the upper-left corner of the PLAYER when it should be the corner of the VIDEO.

I think that this problem can have caused issues in the following : https://github.com/SCENE-CE/mirador-annotation-editor/issues/90****

**Comment by geourjoa** [comment]
I have tested on Mirador Video alone, Mirador 4 with MAE and on Arvest Dev (MMU so MV with MAEV))

**Comment by daxid** [comment]
Well, the annotation is not appearing in the side panel probably because the `motivation` is off (I thought that `painting` was the one to have it displayed in the side panel. I must have been mistaken. You can always change it in https://filebrowser.tetras-libre.fr/).

The annotation still appears on mouse over :

![image](https://github.com/user-attachments/assets/8ba76654-4ac3-459f-ad2f-e62a3e03db88)

**Comment by geourjoa** [comment]
Ok on mouse over I See it

**Comment by geourjoa** [comment]
![image](https://github.com/user-attachments/assets/8a836a0e-30eb-412a-b531-617111c61f10)

SOme progress !

**Comment by daxid** [comment]
Quick progress indeed !! :-)

**Comment by geourjoa** [comment]
Do you have landscape video manifest with annotation ?

**Comment by daxid** [comment]
There you go :

https://files.tetras-libre.fr/dev/cats_video_with_annot.json

Same as before, the annotation only appears on mouseover

**Comment by geourjoa** [comment]
Additional inputs

The mst important file is `src/components/VideoViewer.js`

This file hold the component displaying the video.

VideoViewer.js
![image](https://github.com/user-attachments/assets/0980eac6-87e5-496e-bb5c-ba3f10324fda)

- `styledContainer` : Contains video and his controller. Blue border 
- `StyledVideoContent` Contains video and the annotations layer (`AnnotationsOverlayVideo`).. Red border
- `StyledVideo` Only the video. Pink border
- `AnnotationsOverlayVideo` Layer of annotation. Orange border
- `WindowCanvasNavigationControlsVideo` Controls of the video

**Comment by geourjoa** [comment]
Case 1 

![2024-10-09_14-57](https://github.com/user-attachments/assets/65a336f4-ff16-458a-811f-6f249fa33f19)

**Comment by geourjoa** [comment]
Case 2

![2024-10-09_15-00](https://github.com/user-attachments/assets/8797f1b7-d36e-4b5b-b854-d16afcb44555)

**Comment by geourjoa** [comment]
The annotation (on left top corner) is only displayed on mouse over and with annotation panel opened

**Comment by geourjoa** [comment]
https://github.com/SCENE-CE/mirador-video/pull/6

**Comment by geourjoa** [comment]
Minimal React component showing a video and an overlay div 
@daxid 

```js
import React from 'react';
import ReactPlayer from 'react-player';
import './App.css';

const App = () => {

  const paused = true;

  const playerRef = React.useRef(null);

  const video = {
    id: 'https://youtu.be/dbX5Is_M4CE'
  };

  return (
    <div
    className="outerContainer"
    style={{
      display: 'flex',
      height: '100%',
      justifyContent: 'center',
      overflow: 'auto',
      position: 'relative',
      width: '100%',
    }}
  >
    {video && (
    <>
      <div style={{
        alignItems: 'center',
        backgroundColor: 'black',
        display: 'flex',
        flexDirection: 'column',
        marginBottom: '122px', // TODO Space for navigation controls
        position: 'relative',
        width: '100%',
      }}
      >
        <div style={{
          height: '100%',
          position: 'relative',
          display: 'flex',
          width: '100%',
          alignItems: 'center',
          flexDirection: 'column',
        }}
        >
          <ReactPlayer
            ref={playerRef}
            url={video.id}
            controls={false} // Hide default controls
            pip={false}
            playbackRate={1}
            playing={!paused}
            config={{
              peertube: {
                controls: 0,
                mode: 'p2p-media-loader',
              },
            }}
            style={{
              position: 'relative', // 'absolute' or 'block
              maxWidth: '100%',
              height: '100%',
              textAlign: 'center',
            }}
          />


          <div style={{height: "100%", width: "100%"}}>
            <div style={{backgroundColor: 'red', 
              opacity : '0.3', 
              height: "100%", 
              width: "100%", 
              zIndex: 100, 
              position: "absolute", 
              top:0, left:0}}>

            </div>
          </div>
        </div>
      </div>
      {/* <WindowCanvasNavigationControlsVideo windowId={windowId} /> */}
    </>
    )}
  </div>
  );
};

export default App;
```

**Comment by geourjoa** [comment]
Problem of playback are finally solved 

Available inside MV 1.0.10 package.

Note : to facilitate debug you can add debug config in the Mirador Config.
https://github.com/SCENE-CE/mirador-video/blob/30f1e3b95f24c8998916952ebf25204764bbc6a5/__tests__/integration/mirador/index.html#L31
